.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Summary
```

Comment:
@marlowpayne Does it make sense to even have an issue template for this repo then?  Maybe a README inside the `.github` repo would help to clarify what the files in this folder are for?